services/dispatcher/dispatcher.cpp

```diff
@@ -195,96 +198,83 @@ namespace services::dispatcher {
     }
 
     void dispatcher_t::execute_ql_finish(components::session::session_id_t& session, const result_t& result) {
+        result_storage_[session] = result;
```

Comment:
@kotbegemot @makeyavelly I've added this to be able to unlock `wrapper_dsipathcer` form `wal_success`
```
        auto result = result_storage_[session];
        actor_zeta::send(session_obj.address(), dispatcher_t::address(), handler_id(route::execute_ql_finish), session, result);
```
I don't know how effective is it.